src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.commands;
+
+
+import java.util.List;
+
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Sorts and list all persons in the address book according in lexicographical order.
+ */
+public class SortCommand extends Command{
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the lexicographical order in the address book as a list with index numbers.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        /*List<String> personsName = new ArrayList<String>();
```

Comment:
Delete unnecessary code instead of commenting it out. Using Git allows you to rollback to previous code without effort.